recipes/glimpse-bio/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "v1.1.1" %}
+
+package:
+  name: glimpse-bio
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/odelaneau/GLIMPSE/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ae3b4349c02f589022d5f22083f81e7a82dc93daf8c29d9beeeb7f8d34f80465
+  patches:
+    - 0001-fix-string_utils.h-warning.patch
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - htslib
+    - bzip2
+    - zlib
+    - boost
+    - boost-cpp
+    - openssl
+    - pthread-stubs
+  run:
+    - htslib
+    - bzip2
+    - zlib
```

Comment:
you can remove zlib, bzip2 and htslib I think from the run deps

Reply:
Happy to do so, but curious why so I can learn for my next recipe.

Reply:
For libraries, we can define `run_exports`, so this is implicit: https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L13

It just cleans up recipes on your side and makes sure it gets really exported, every time, on the global ecosystem.